client/planning-extension/src/assignments-overview/index.tsx

```diff
@@ -0,0 +1,245 @@
+                ),
+        );
+
+        Promise.all([
+            refetchDesks ? fetchDesks() : Promise.resolve(state.desks),
+            refetchContentTypes ? fetchContentTypes() : Promise.resolve(state.contentTypes),
+            refetchAssignments ? fetchAssignments(state.currentUser._id) : Promise.resolve(state.assignments),
+        ]).then(([desks, contentTypes, assignments]) => {
+            this.setState({
+                loading: false,
+                desks: desks,
+                contentTypes: contentTypes,
+                assignments: assignments,
+            });
+        });
+    }
+
+    componentDidMount() {
+        superdesk.session.getCurrentUser()
+            .then((currentUser) =>
+                Promise.all([
+                    fetchDesks(),
+                    fetchContentTypes(),
+                    fetchAssignments(currentUser._id),
+                ]).then(([desks, contentTypes, assignments]) => {
+                    this.setState({
+                        loading: false,
+                        currentUser: currentUser,
+                        desks: desks,
+                        contentTypes: contentTypes,
+                        assignments: assignments,
+                    });
+                })
+            );
+    }
+
+    componentWillUnmount() {
+        this.eventListenersToRemoveBeforeUnmounting.forEach((removeListener) => {
+            removeListener();
+        });
+    }
+
+    render() {
+        if (this.state.loading === true) {
+            return null;
+        }
+
+        const {assignments, desks, contentTypes} = this.state;
+        const itemsCount = assignments.length;
+        const grouped = groupBy(assignments, (item) => item.assigned_to.desk);
+
+        return (
+            <DropdownTree
+                groups={Object.keys(grouped).map((deskId) => ({
+                    render: () => (
+                        <GroupLabel>
+                            <Badge type="highlight" text={grouped[deskId].length.toString()} />
+                            <span style={{marginLeft: 6}}>{desks[deskId].name}</span>
+                        </GroupLabel>
+                    ),
+                    items: grouped[deskId],
+                }))}
+                getToggleElement={
+                    (isOpen, onClick) => (
+                        <TopMenuDropdownButton
+                            onClick={() => {
+                                if (itemsCount > 0) {
+                                    onClick();
+                                }
+                            }}
+                            active={isOpen}
+                            disabled={itemsCount < 1}
+                            pulsate={false}
+                            data-test-id="toggle-button"
+                        >
+                            <Badge type="highlight" text={itemsCount.toString()}>
+                                <span style={{color: isOpen ? '#3783A2' : 'white'}}>
```

Comment:
Would this work better `{color: isOpen ? 'var(--sd-slugline-color)' : 'var(--color-bg-00)'}`. 

Using vars from the ui-framework allows the framework to control these colours (especially in the case of accessibility / dark-ui)